components/governance/org.wso2.carbon.governance.registry.extensions/src/org/wso2/carbon/governance/registry/extensions/indexers/RXTIndexer.java

```diff
@@ -112,4 +118,27 @@ public void setQName(QName qName) throws GovernanceException {
             return attributes;
         }
     }
+
+    /**
+     * This method is used to update rxt unbounded fields map.
+     *
+     * @param elementString     rxt configuration.
+     * @throws RegistryException
+     */
+    private static void updateTenantsUnboundedFieldMap(String elementString) throws
+            RegistryException {
+        int tenantId = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantId();
+        RxtUnboundedEntryBean rxtUnboundedFields = RxtUnboundedDataLoadUtils.getRxtUnboundedEntries(
+                elementString);
+        if (rxtUnboundedFields != null) {
+            Map<String, List<String>> currentTenantUnboundedFields = RxtUnboundedFieldManagerService.getInstance()
+                    .getTenantsUnboundedFields().get(tenantId);
+            if (currentTenantUnboundedFields == null) {

```

Comment:
is it possible it check isEmpty here?[1]

[1] - https://docs.oracle.com/javase/7/docs/api/java/util/Map.html#isEmpty()